pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -572,16 +574,18 @@ func (cloud *CloudProvider) insertRegionalDisk(
 			klog.Warningf("GCE PD %s already exists, reusing", volKey.Name)
 			return nil
 		}
-		return status.Error(codes.Internal, fmt.Sprintf("unknown Insert disk error: %v", err.Error()))
+		return fmt.Errorf("unknown Insert Regional disk error: %w", err)
```

Comment:
So the idea here is to pass whatever the wrapped err is, which should be correct w/rt to temporary or not, rather than converting to internal?

Reply:
this is sort of unrelated to the main objective of this PR, I noticed that `insertZonalDisk` function has this same error thrown as `fmt.Errorf("unknown Insert Regional disk error: %w", err)` while this function does the manual code, I think it make sense to align the two and it seems more appropriate to do the former.

let me know what you think?

Reply:
+1 to alignment

Reply:
and +1 to not changing the code to internal.

Reply:
Add a comment to the effect that this may or may not be retried depending on the error code.

Reply:
done